src/components/sections/ProductGallery/product-gallery.scss

```diff
@@ -4,12 +4,27 @@
   --product-listing-row-height: var(--space-6);
 
   height: 100%;
+  padding-top: 0;
```

Comment:
This seems to be needed for the search page, but the existing padding is needed for PLP. 
😕 

https://user-images.githubusercontent.com/381395/158908807-66f4f054-4c76-4670-9730-c5a05529cb25.mp4

![CleanShot 2022-03-17 at 20 09 31](https://user-images.githubusercontent.com/381395/158908842-470ccf14-f87b-458f-b71e-dd23a270374b.png)

Now how to solve this only for the search page... 
🤔

Reply:
After aligning with the designers we changed the design a bit to accomodate the title gracefully, not interfering with the PLP style.

Reply:
After aligning with the designers we changed the design a bit to accommodate the title gracefully, not interfering with the PLP style.